andela/hc-kalii
# #153727991 Users priorty list

#### What does this PR do?
* Adds specifications to the project that enable notifications to be sent to members of a specific team, based on the priority of the member in that team

#### Description of Task to be completed?
* write tests to check member priority
* update the members model by adding priority field
* add update priority form
*  write methods to handle change of member's priority
*  create update priority form widget to the HTML

 #### How should this be manually tested?
* Execute `./manage.py runserver`  then `./manage.py ensuretriggers` then`./manage.py sendalerts`. in separate terminals
* Login with a valid email address and confirm login from your Inbox.
* Go to account settings and in Team Access, set the team name and Invite a member
   and then set priority.

#### What are the relevant pivotal tracker stories?
* [153727991](https://www.pivotaltracker.com/story/show/153727991)
